### PR TITLE
Update cyclic build dependencies

### PR DESCRIPTION
**Pull request by nickpape-msft**
* Completely remove old versions of Growl from our shrinkwrap

### CODE REVIEW

**Comment by pgonzal** [comment]
Your build failed

File: core-build/gulp-core-build-karma/gulpfile.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 let build = require('@microsoft/node-library-build');
-
+build.mocha.enabled = false;
 build.initialize(require('gulp'));
```
Comment (iclanton) [inline core-build/gulp-core-build-karma/gulpfile.js]: Why do you have to disable mocha everywhere?

Comment (nickpape-msft) [reply core-build/gulp-core-build-karma/gulpfile.js]: Something changed when I updated mocha and basically it fails if it can't find any tests.

**Comment by iclanton** [comment]
Approved.

**Pull request merged**